vms/evm/subscriber.go

```diff
@@ -26,40 +30,84 @@ const (
 	maxClientSubscriptionBuffer = 20000
 	subscribeRetryTimeout       = 1 * time.Second
 	maxResubscribeAttempts      = 10
+	MaxBlocksToProcess          = 200
 )
 
 var (
 	// Errors
 	ErrInvalidLog = errors.New("invalid warp message log")
 )
 
+// The filter query used to match logs emitted by the Warp precompile
```

Comment:
```suggestion
// The filter query used to match logs emitted by the Warp precompile.
```

Reply:
Actually, looks like we haven't been 100% consistent with punctuation to date. I don't have strong preference either way, but lets decide on a convention.

Reply:
Agreed. I'm going to leave the comments as is for now, then circle back across the whole repo.